azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKInitializationWrapper.kt

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.service.sdk
+
+import android.content.Context
+import androidx.core.util.Consumer
+import com.azure.android.communication.calling.CallAgent
+import com.azure.android.communication.calling.CallAgentOptions
+import com.azure.android.communication.calling.CallClient
+import com.azure.android.communication.calling.CallClientOptions
+import com.azure.android.communication.calling.IncomingCall
+import com.azure.android.communication.calling.IncomingCallListener
+import com.azure.android.communication.common.CommunicationTokenCredential
+import com.azure.android.communication.ui.calling.DiagnosticConfig
+import com.azure.android.communication.ui.calling.logger.Logger
+import com.azure.android.communication.ui.calling.service.sdk.ext.setTags
+import java9.util.concurrent.CompletableFuture
+
+internal object CallingSDKInstanceManager {
+    var callingSDKCallAgentWrapper: CallingSDKCallAgentWrapper? = null
+}
+
+internal class CallingSDKCallAgentWrapper(private val logger: Logger) {
```

Comment:
Some internal developer documentation would be useful here on the class.

Why is it used, what is it's purpose. Etc.

Reply:
As per discussion we will fix for GA.